### ISSUE TITLE
Why LeJEPA is considered part of the JEPA family?

### ISSUE THREAD

**Issue by ForestsKing** [rbalestr-lab/lejepa#13]
Hello, I am a beginner in JEPA and would greatly appreciate some guidance. I am trying to understand what exactly qualifies a model as a JEPA. LeJEPA, to me, seems quite similar to traditional contrastive learning methods (e.g., [1]): one loss pulls carefully constructed augmented views closer together, while another loss encourages embedding uniformity. Could someone kindly clarify the core difference between LeJEPA and [1], and why LeJEPA is considered part of the JEPA family? Additionally, I understand that I-JEPA can operate without data augmentations (aside from cropping). Does LeJEPA also share this characteristic? Thank you very much for your help!

[1] Understanding Contrastive Representation Learning through Alignment and Uniformity on the Hypersphere, ICML 2020

**Comment by RandallBalestriero** [comment]
That's a great question! As pointed out by @kadarsh22, the main question for any non-contrastive method is how to prevent collapse. This is our main contribution: we introduce SIGReg as a regularizer to enforce isotropic Gaussian embeddings. That regularizer can be used in any non-contrastive method (JEA, JEPA, ...). Now whether you want a predictor or not really depends on your setup. In our case we have symmetric views and we don't condition our predictor on any extra side-information. So as per our ablation, having a predictor or not doesn't impact final performance (you learn the same representation). If we were to condition the predictor e.g. to the data-augmentation realized parameters, then results would be different. If you deal with videos and have access to frames and actions (governing the transition between frames), then an action-conditioned predictor would be recommended (especially if you want to do planning post-training). 

You can perfectly take the prediction task of I-JEPA, and replace the teacher-student mechanism with SIGReg, in which case you would likely maintain same final performances (or maybe even improve them?) while removing a bunch of hyper-parameters :) LeJEPA also works with cropping only, I didn't include those extra ablations but from all the current DAs, you can keep RRC and color jitter only + do patch masking (like MAE/IJEPA) to still reach almost the same final performance. Let me know if this doesn't answer your question!

**Comment by ForestsKing** [comment]
Thank you for your response, and thanks as well to @kadarsh22 for the helpful explanation!

My understanding is that [1] prevents collapse by encouraging pairwise uniformity among embeddings within a batch, while sigreg achieves a similar goal by pushing batch embeddings toward a normal distribution. But in essence, both methods seem to fail when the batch size is 1 (please correct me if I’m misunderstanding). In that sense, both approaches implicitly treat other samples in the batch as negative examples. This may explain why both methods can work in a heuristics-free setting without stop-gradient or teacher–student architectures. I tried replacing the sigreg_loss with the loss from [1]. Although the model still converges without collapsing, the kNN accuracy drops significantly, which suggests the superiority of sigreg.

Additionally, if I understand you correctly, are you saying that it is possible to implement I-JEPA without a teacher–student setup by using a single encoder to produce both target_embed and context_embed, and then using a predictor to predict target_embed from context_embed? Have you conducted any experiments along these lines? If so, I would be very interested in the results.
